bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -163,6 +174,10 @@ spec:
                 {{- end }}
             {{- end }}
             {{- if .Values.auth.tls.enabled }}
+            {{- if .Values.auth.tls.autoGenerated }}
+            - name: KEYCLOAK_TLS_CREATE_KEYSTORE
+              value: "true"
+            {{- end }}
```

Comment:
Minor thing: This can also be achieved with a `ternary`

Reply:
+1

Reply:
Understood, I will apply the change.
Thanks for the suggestion!